front-end/src/views/login/index.vue

```diff
@@ -127,6 +133,12 @@ export default {
         this.passwordType = 'password'
       }
     },
+    handleMessage(event) {
```

Comment:
Is this method useful?
If it is not used, then I suggest to remove it

Reply:
Yes, in the next pr, this function will be used in GitHub login to communicate in two different windows at the front end to receive some information called back by Github and ensure the success of user authentication.